sensors/triggers/params.go

```diff
@@ -179,12 +200,11 @@ func ResolveParamValue(src *v1alpha1.TriggerParameterSource, events map[string]*
 			} else {
 				// Default value doesn't exist so return the whole event payload
 				eventPayload, err = json.Marshal(&event)
+				if err == nil {
```

Comment:
Why moved here? It changed the behavior. when `src.Value != nil`, it originally returns directly, but now it does not.

Reply:
reverted